app/src/main/java/at/aau/edu/appdev/messenger/ui/chat/MessageAdapter.kt

```diff
@@ -29,13 +31,13 @@ class MessageAdapter : ListAdapter<Message, MessageViewHolder>(Differ()) {
             R.layout.item_message -> {
                 val binding = ItemMessageBinding.inflate(inflater, parent, false)
 
-                MessageViewHolder.TextViewHolder(binding)
+                MessageViewHolder.TextViewHolder(binding, context)
```

Comment:
just as info, `binding` has a view (`root`). A View always is attached to a context, so no need to pass one here.

Reply:
good to know - will correct this in my current working branch